src/openrct2-ui/input/MouseInput.cpp

```diff
@@ -1463,6 +1463,11 @@ void InputStateWidgetPressed(
         }
         return;
     }
+    else if (_inputState == InputState::DropdownActive && gDropdownIsColour)
```

Comment:
```suggestion
    else if (gDropdownIsColour)
```

It can only be `DropdownActive`, since the `if` above covers all other cases, no?

Reply:
I find the inverted logic hard to read, I'll change it maybe someone can refactor that afterwards and do it the other way around.